workspaces/ballerina/ballerina-visualizer/src/views/BI/HelperPaneNew/Views/Variables.tsx

```diff
@@ -71,9 +71,11 @@ const VariableItem = ({ item, onItemSelect, onMoreIconClick }: VariableItemProps
                     event.stopPropagation();
                     onMoreIconClick(item.label);
                 }}>
-                    <VariableTypeIndicator >
-                        {item.description}
-                    </VariableTypeIndicator>
+                    <Tooltip content={item.description} position="top">
+                        <VariableTypeIndicator >
+                            {item.description.length > 5 ? item.description.slice(0, 5) + '…' : item.description}
+                        </VariableTypeIndicator>
```

Comment:
We should be able to handle this with text-overflow styling instead of slice based on the length. can you please check